server/src/child.rs

```diff
@@ -58,6 +60,8 @@ pub enum DebugTrace {
     Breakpoint(usize),
     Exit(u32),
     Crash,
+    StopRequest,
```

Comment:
A better name than `StopRequest` might be `Cancel`, to match `TraceData::Cancel`.